web_monitoring/differs.py

```diff
@@ -51,11 +52,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    obj = web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
```

Comment:
I think the plan is to retire go-to-diff, long term, but in general we do want a path to integrate with differs written in other languages, and go-to-diff might be a nice proof of concept for that. But I don't think that's a "version 0" priority.